api/proto/teleport/legacy/types/types.proto

```diff
@@ -1578,6 +1578,10 @@ message ClusterNetworkingConfigSpecV2 {
     (gogoproto.jsontag) = "proxy_ping_interval,omitempty",
     (gogoproto.casttype) = "Duration"
   ];
+
+  // AssistCommandExecutionWorkers determines the number of workers that will
+  // execute arbitrary remote commands on servers (e.g. through Assist) in parallel
+  int64 AssistCommandExecutionWorkers = 11 [(gogoproto.jsontag) = "assist_command_execution_workers,omitempty"];
```

Comment:
nit: The value cannot be negative and probably having 2B+ runners is not wise. I'd replace it with `uint32` so we don't need to worry about `< 0` values

Reply:
My 2c: I'd go with `int32` and explicitly validate against negatives. Not all proto supported languages maps unsigned types well, so I've grown a habit of avoiding them.